pkgs/by-name/la/lammps/package.nix

```
@@ -44,9 +45,21 @@
   extraCmakeFlags ? { },
   # Extra `buildInputs` - meant for packages that require more inputs
   extraBuildInputs ? [ ],
+  overrideStdenv ? null,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+let
+  # Ease adding CUDA support. See:
+  # https://github.com/lammps/lammps/issues/4420
+  stdenv' =
+    if !(builtins.isNull overrideStdenv) then
+      overrideStdenv
```

I would rather make `overrideStdenv` a bool:
```suggestion
    if  overrideStdenv then
      stdenv
```

---

> I would rather make `overrideStdenv` a bool:

Good idea.